docs/src/main/asciidoc/hibernate-search-elasticsearch.adoc

```diff
@@ -537,8 +537,8 @@ quarkus.hibernate-orm.sql-load-script=import.sql <4>
 
 quarkus.hibernate-search.elasticsearch.version=7 <5>
 quarkus.hibernate-search.elasticsearch.analysis.configurer=org.acme.hibernate.search.elasticsearch.config.AnalysisConfigurer <6>
-quarkus.hibernate-search.elasticsearch.index-defaults.lifecycle.strategy=drop-and-create <7>
-quarkus.hibernate-search.elasticsearch.index-defaults.lifecycle.required-status=yellow <8>
+quarkus.hibernate-search.schema-management.strategy=drop-and-create <7>
+quarkus.hibernate-search.elasticsearch.index-defaults.schema-management.required-status=yellow <8>
```

Comment:
Could you write something I would include in the migration guide?

Reply:
@gsmet Here you go:

``````
## Hibernate Search + Elasticsearch (Preview)

Configuration properties relative to Elasticsearch index lifecycle management have changed.
See [this section of the reference documentation](https://docs.jboss.org/hibernate/search/6.0/reference/en-US/html_single/#mapper-orm-schema-management) for detailed documentation about schema management.

In short:

* `quarkus.hibernate-search.elasticsearch.index_defaults.lifecycle.strategy` is now `quarkus.hibernate-search.schema-management.strategy` and accepts the same values as before: `none`, `create`, `validate`, `update`, ...
* A new schema management strategy on startup was introduced: `create-or-validate`. It create indexes if they don't exist, validates their schema if they already exist.
* The default schema management strategy on startup is now `create-or-validate`.
*  `quarkus.hibernate-search.elasticsearch.index_defaults.lifecycle.required_status` is now `quarkus.hibernate-search.elasticsearch.index_defaults.schema_management.required_status`.
*  `quarkus.hibernate-search.elasticsearch.index_defaults.lifecycle.required_status_wait_timeout` is now `quarkus.hibernate-search.elasticsearch.index_defaults.schema_management.required_status_wait_timeout`.

Also related: you can now use [`SearchSchemaManager` APIs](https://docs.jboss.org/hibernate/search/6.0/reference/en-US/html_single/#mapper-orm-schema-management-manager) to trigger schema management operations at the time of your choosing (not just on startup).
``````